### PR TITLE
Task/sscs 3227 tribunals api docker

### PR DESCRIPTION
**Pull request by savirneni**
New dockerfile added for the DockerHub.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-COPY build/install/tribunals-case-api /opt/app/
+COPY build/libs/tribunals-case-api.jar /opt/app/
 
 WORKDIR /opt/app
 
-HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" curl --silent --fail http://localhost:8083/health
+HEALTHCHECK NONE
 
-EXPOSE 8083
+EXPOSE 8082
```
Comment (sathishbabuj) [inline Dockerfile]: sscs case loader uses port 8082. So it will be better to use TCA API default port of 8080

File: hub.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+FROM openjdk:8-jre-alpine
+
+COPY --from=builder /home/gradle/src/build/libs/tribunals-case-api.jar /opt/app/
+
+WORKDIR /opt/app
+
+HEALTHCHECK NONE
+
+EXPOSE 8082
```
Comment (sathishbabuj) [inline hub.Dockerfile]: sscs case loader uses port 8082. So it will be better to use TCA API default port of 8080

File: Dockerfile (hunk 1 above)
Comment (savirneni) [reply Dockerfile]: Port updated to use 8080

File: hub.Dockerfile (hunk 2 above)
Comment (savirneni) [reply hub.Dockerfile]: Port updated to use 8080

**Pull request merged**